sample.env

```diff
@@ -23,6 +23,9 @@ SERVICEBOT_ID=servicebot_id_from_servicebot_dashboard
 SERVICEBOT_HMAC_SECRET_KEY=secret_key_from_servicebot_dashboard
 
 PAYPAL_SUPPORTERS=1703
+PAYPAL_CLIENT_ID=id_from_paypal_dashboard
+PAYPAL_SECRET=secret_from_paypal_dashboard
+PAYPAL_FIVE_DOLLAR_MONTHLY_PLAN_ID=subscription_plan_id_from_paypal_dashboard
```

Comment:
does the subscription id require us to keep it secret? If not, whats the best place to keep it?

Reply:
the subscription id changes between sandbox and live. so I thought it would better to keep it in env where we can change it easily for dev and production

Reply:
See https://github.com/freeCodeCamp/freeCodeCamp/pull/38239#discussion_r384335418